### PR TITLE
Audio/Video Support

### PR DESCRIPTION
**Pull request by markgrossnickle**
Audio/Video is working on PC/Mac. Tested on my mac (screen, camera, and texture). Use the enum dropdown in LiveKitTester.cs.  

You can set autosubscribe to false but then will need to SetSubscribedForRemote(true/false) to manage your own subscriptions.

### CODE REVIEW

File: Runtime/Scripts/Internal/FFIClient.cs [hunk 1]
```diff
@@ -199,24 +208,26 @@ public FfiResponse SendRequest(FfiRequest request)
...
-            Utils.Debug("Callback... " + response.ToString());
             // Run on the main thread, the order of execution is guaranteed by Unity
             // It uses a Queue internally
             Instance.context?.Post((resp) =>
             {
                 var r = resp as FfiEvent;
-                if (r?.MessageCase != FfiEvent.MessageOneofCase.Logs)
-                    Utils.Debug("Callback: " + r?.MessageCase);
+                //if (r?.MessageCase != FfiEvent.MessageOneofCase.Logs)
+                //    Utils.Debug("Callback: " + r?.MessageCase);
```
Comment (aixaCode) [inline Runtime/Scripts/Internal/FFIClient.cs:226]: I guess this and other commented logs are still WIP?

File: Runtime/Scripts/Rooms/Participants/Factory/IParticipantFactory.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 using LiveKit.Internal;
 using LiveKit.Proto;
 using LiveKit.Rooms.TrackPublications;
+using UnityEngine;
```
Comment (aixaCode) [inline Runtime/Scripts/Rooms/Participants/Factory/IParticipantFactory.cs:6]: Do we need this namespace?

File: Runtime/Scripts/Rooms/TrackPublications/TrackPublication.cs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 using LiveKit.Internal.FFIClients.Requests;
 using LiveKit.Proto;
 using LiveKit.Rooms.Tracks;
+using UnityEngine;
```
Comment (aixaCode) [inline Runtime/Scripts/Rooms/TrackPublications/TrackPublication.cs:4]: Looks like it's not needed as well

**Review by aixaCode — APPROVED** [review]

File: Runtime/Scripts/Internal/FFIClient.cs (hunk 1 above)
Comment (markgrossnickle) [reply Runtime/Scripts/Internal/FFIClient.cs:226]: This one is just very spammy but I wasn't sure if we needed it or not. Thoughts?

File: Runtime/Scripts/Internal/FFIClient.cs [hunk 4]
```diff
@@ -245,7 +256,7 @@ static unsafe void FFICallback(IntPtr data, int size)
                         Instance.AudioStreamEventReceived?.Invoke(r.AudioStreamEvent!);
                         break;
                     case FfiEvent.MessageOneofCase.CaptureAudioFrame:
-                        Utils.Debug(r.CaptureAudioFrame!);
+                        //Utils.Debug(r.CaptureAudioFrame!);
```
Comment (markgrossnickle) [inline Runtime/Scripts/Internal/FFIClient.cs:259]: This one is not needed

**Pull request merged**